packages/slider/src/Slider.ts

```diff
@@ -124,52 +120,33 @@ export class Slider extends Focusable {
         `;
     }
 
-    protected updated(changedProperties: PropertyValues): void {
-        if (changedProperties.has('value')) {
-            if (this.value === this.input.valueAsNumber) {
-                this.dispatchInputEvent();
-            } else {
-                this.value = this.input.valueAsNumber;
-            }
-        }
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.handleController.hostConnected();
     }
 
-    private renderLabel(): TemplateResult {
-        return html`
-            <div id="labelContainer">
-                <label id="label" for="input"><slot>${this.label}</slot></label>
-                <output id="value" aria-live="off" for="input">
-                    ${this.ariaValueText}
-                </output>
-            </div>
-        `;
+    public disconnectedCallback(): void {
+        super.disconnectedCallback();
+        this.handleController.hostDisconnected();
     }
 
-    private renderTrackLeft(): TemplateResult {
-        if (this.variant === 'ramp') {
-            return html``;
-        }
-        return html`
-            <div
-                class="track"
-                id="track-left"
-                style=${styleMap(this.trackStartStyles)}
-                role="presentation"
-            ></div>
-        `;
+    public update(changedProperties: Map<string, boolean>): void {
+        this.handleController.hostUpdate();
+        super.update(changedProperties);
     }
 
-    private renderTrackRight(): TemplateResult {
-        if (this.variant === 'ramp') {
-            return html``;
-        }
+    private renderLabel(): TemplateResult {
         return html`
-            <div
-                class="track"
-                id="track-right"
-                style=${styleMap(this.trackEndStyles)}
-                role="presentation"
-            ></div>
+            <div id="labelContainer">
+                <label id="label" for="input"><slot>${this.label}</slot></label>
+                ${this.hideValueLabel
+                    ? ''
+                    : html`
+                          <output id="value" aria-live="off" for="input">
```

Comment:
We should use something like the following on this instead here:
```
.visually-hidden {
  clip: rect(0 0 0 0);
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}
```
This will allow the output to still be available for the screen reader.

Reply:
I've often thought to add this to our `styles` package and then ended up not fully needing the rule in the project I was working on, for instance.